pkg/resources/test/fixtures/deployment-openstack-1.25.0-openstack-cloud-controller-manager-externalCloudProvider.yaml

```diff
@@ -38,7 +38,7 @@ spec:
         env:
         - name: SSL_CERT_FILE
           value: /etc/kubermatic/certs/ca-bundle.pem
-        image: docker.io/k8scloudprovider/openstack-cloud-controller-manager:v1.25.4
+        image: docker.io/k8scloudprovider/openstack-cloud-controller-manager:v1.25.5
```

Comment:
A bit unrelated to your changes but can we please switch the registry here i.e switch image to `registry.k8s.io/provider-os/openstack-cloud-controller-manager:v1.25.5`

Reply:
Not really just unrelated, but necessary :-D The 1.27.x images do not seem to be published on docker.io anymore, so #12230 started to fail.